### ISSUE TITLE
Add map option for destinations 

### ISSUE THREAD

**Issue by mrjkey** [Tris20/FairFareFinder#13]
Could be nice to view the destinations in a map instead of a table. I don't know where many places are

**Comment by Tris20** [comment]
Agree. The hyperlink in table of the cityname is currently for the location. We could embed this with google maps instead.
